javascript-objects/main.js

```diff
@@ -0,0 +1,27 @@
+const student = { firstName: 'Jeremy', lastName: 'Nguyen', age: 27 };
+const fullName = student.firstName + ' ' + student.lastName;
+
+console.log('value of fullName: ' + fullName);
+
+student.livesInIrvine = true;
+student.previousOccupation = 'Production Assistant';
+
+console.log('value of student.livesInIrvine: ' + student.livesInIrvine);
+console.log(
+  'value of student.previousOccupation: ' + student.previousOccupation
+);
+console.log('value of student: ', student);
+
+const vehicle = { make: 'Tesla', model: 'Y', year: 2023 };
+
+vehicle['color'] = 'white';
+vehicle['isConvertible'] = 'no';
+
+console.log('value of vehicle: ', vehicle);
+
+const pet = { name: 'Austin', type: 'dog' };
+
+delete pet.name;
+delete pet.type;
+
+console.log('value of pet: ' + pet);
```

Comment:
Do you see how in your console, you're getting `value of pet: [object, Object]`? This is exactly why we don't want to do console logs with the `+` operator, and we want to use a `,` to separate them instead. I will touch on this behavior in our Q&A.